common/flatpak-dir-private.h

```diff
@@ -32,7 +32,7 @@
  * The version field was added in flatpak 1.2, anything before is 0.
  *
  * Version 1 added appdata-name/summary/version/license
- * Version 2 added extension-of
+ * Version 2 added extension-of/appdata-content-rating
```

Comment:
We did release ver 2 in 1.5.0, but that is an unstable release, so i'm fine with reusing the same version here.

Reply:
You hadn’t released 1.5.0 at the time I wrote that!